### PR TITLE
Add FAQ for non-profit program

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Review by aspleenic — APPROVED** [review]
Looks good. Do we look to have a waiver for folks who can't who can't afford the $250 with certain cases?

**Review by BenLloydPearson — APPROVED** [review]
@jasonblais I streamlined the content and removed info about cost. I also added a few of the requirements we previously spoke about.

**Comment by jasonblais** [comment]
@BenLloydPearson Was the free license approved by MLT?

We got an exception to onboard 20 communities to Mattermost Cloud for free, but I wasn't aware that any other changes to our non-profit license had been approved by MLT.

**Comment by BenLloydPearson** [comment]
Can we start with that 20 and expand from there if necessary?

**Comment by jasonblais** [comment]
@BenLloydPearson What are your thoughts on below? It removes mention of the price, since I'm not aware we've gotten sign off on a free nonprofit license beyond the 20 Cloud communities.

-------

Yes. The Mattermost Nonprofit License enables foundations and other non-profit organizations that support open-source projects or other technology initiatives to apply the benefits of Mattermost Enterprise Edition to advancing their missions with special non-profit pricing.

An eligible organization needs to be an official nonprofit or charity, and also non-government, non-academic and non-commercial in nature, with no religious affiliation and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing-self-managed/>`_ instead.

Organizations that receive a Mattermost non-proft license must make their server publically accessible for anyone to sign up and join. Mattermost will also have the right to place the name and logo of the nonprofit or charitable institution on our website.

Please contact us at community[at]mattermost.com to inquire about the Mattermost Nonprofit License.

**Comment by BenLloydPearson** [comment]
LGTM!

**Comment by jasonblais** [comment]
Queuing for editor review @justinegeffen

**Review by justinegeffen — COMMENTED** [review]
Some minor typos and making "nonprofit" consistent. FWIW, both non-profit and nonprofit are accepted, however nonprofit is the American version. :)

**Review by justinegeffen — APPROVED** [review]

File: source/about/licensing-and-subscription.rst [hunk 1]
```diff
@@ -247,7 +247,13 @@ If you don't plan to renew your Enterprise Edition subscription, revoke the expi
...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Yes, please see the `Mattermost Monday License program <https://mattermost.com/nonprofit/>`_.
+Yes. The Mattermost Nonprofit License enables foundations and other nonprofit organizations that support open source projects or other technology initiatives to apply the benefits of Mattermost Enterprise Edition to advancing their missions with special nonprofit pricing.
+
+To be eligible, an organization needs to be an official nonprofit or charity; as well as non-government, non-academic and non-commercial in nature, with no religious affiliation; and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing-self-managed/>`_ instead.
+
+Organizations that receive a Mattermost nonprofit license must make their server publicly-accessible for anyone to sign up and join. Mattermost will also have the right to place the name and logo of the nonprofit or charitable institution on our website.
```
Comment (justinegeffen) [inline source/about/licensing-and-subscription.rst:254]:
```suggestion
Organizations that receive a Mattermost Nonprofit License must make their server publicly-accessible for anyone to sign up and join. Mattermost will also have the right to place the name and logo of the nonprofit or charitable institution on our website.
```

**Comment by jasonblais** [comment]
Thanks Justine!

**Pull request merged**